homeassistant/components/gdacs/const.py

```diff
@@ -0,0 +1,29 @@
+"""Define constants for the GDACS integration."""
+from datetime import timedelta
+
+DOMAIN = "gdacs"
+
+PLATFORMS = ("sensor", "geo_location")
+
+FEED = "feed"
+
+CONF_CATEGORIES = "categories"
+
+DEFAULT_ICON = "mdi:alert"
+DEFAULT_RADIUS = 500.0
+DEFAULT_SCAN_INTERVAL = timedelta(minutes=5)
+
+SIGNAL_DELETE_ENTITY = "gdacs_delete_{}"
+SIGNAL_UPDATE_ENTITY = "gdacs_update_{}"
+SIGNAL_STATUS = "gdacs_status_{}"
+
+SIGNAL_NEW_GEOLOCATION = "gdacs_new_geolocation_{}"
+
+VALID_CATEGORIES = [
```

Comment:
Looks like we can just use the values here instead:
https://github.com/exxamalte/python-aio-georss-gdacs/blob/016dc28d16b95f239feffaa7a627a46331b512ce/aio_georss_gdacs/consts.py#L25-L32

Reply:
Yes, we can.